test/integration/controllers/user.spec.js

```diff
@@ -26,14 +30,24 @@ describe('User controller', () => {
   })
 
   describe(`GET ${endpointUrl}`, () => {
-    it('should GET all users', async () => {
+    it('should return a list of users', async () => {
       let user = await User.create(testUser)
 
-      const response = await app.get(endpointUrl)
-
-      expect(response.statusCode).toBe(200)
-      expect(Array.isArray(response.body)).toBeTruthy()
-      expect(response.body).toEqual(expect.any(Array))
+      const query = {
```

Comment:
why is that here? should it be next to all vars?

Reply:
fixed